pages/about/index.tsx

```diff
@@ -32,6 +33,64 @@ interface TeamSection {
   members: string;
 }
 
+type ArticleType = {
+  title: string;
+  url: string;
+  sourceName: string;
+  lang: string | null;
```

Comment:
`lang: string` should be sufficient, we want to enforce that articles in the JSON file have `lang` specified :)